endpoints/users/controller.py

```diff
@@ -33,17 +34,15 @@
 register_parser.add_argument(
     'password', help='Field password cannot be blank', required=True)
 
-
 class ListUsersResource(Resource):
+    @jwt_required
     def get(self):
         try:
             users = User.query.all()
```

Comment:
No queries in the view

Reply:
Ok, working on building a repository for every model.